test/index.test.js

```diff
@@ -58,4 +58,27 @@ describe('PublicationClient', () => {
       jest.clearAllTimers();
     });
   });
+
+  it('getSubscriptionById', () => {
+    const pub = new PublicationClient('https://127.0.0.1', {
+      lastDataTimeout: 1,
+      paranoid: true,
+    });
+    const subscriptionKey = {
+      name: 'orgs',
+      options: {
+        bootstrap: false,
+        expandMembers: true,
+      },
+    };
+    pub._nextSubscriptionId = 0;
+    for (let i = 0; i < 10; i++) {
+      pub.subscribe(`${subscriptionKey.name}-${i}`, subscriptionKey.options);
+    }
+    const subscriptionId = '5';
+    const subscription = pub.getSubscriptionById(subscriptionId);
```

Comment:
Maybe add one more test to cover https://github.com/mixmaxhq/publication-client/pull/31/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R337 ?